boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp.dts

```diff
@@ -90,22 +42,57 @@
 	load-capacitance-femtofarad = <15000>;
 };
 
-&uart20 {
+&grtc {
+	owned-channels = <0 1 2 3 4 5 6 7 8 9 10 11>;
+	/* Channels 7-11 reserved for Zero Latency IRQs, 3-4 for FLPR */
+	child-owned-channels = <3 4 7 8 9 10 11>;
 	status = "okay";
-	current-speed = <115200>;
-	pinctrl-0 = <&uart20_default>;
-	pinctrl-1 = <&uart20_sleep>;
-	pinctrl-names = "default", "sleep";
 };
 
-&uart30 {
-	current-speed = <115200>;
-	pinctrl-0 = <&uart30_default>;
-	pinctrl-1 = <&uart30_sleep>;
-	pinctrl-names = "default", "sleep";
+&cpuapp_rram {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x0 DT_SIZE_K(64)>;
+		};
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x10000 DT_SIZE_K(324)>;
```

Comment:
why s and ns image have both 324k?

Reply:
This is probably a question to @michalek-no ? #68393 
I did not change the partition layout. I only reduced them proportionally to create some space for the FLPR kernel.